src/KnownTools.ts

```diff
@@ -190,21 +246,36 @@ const knownTools = {
   },
   "elm-json": {
     "0.2.8": {
-      linux: {
-        hash: "d4561a9a2bd70e1a4a4c61e75cac03d3bb1a3b5f99322afa720a2f2d84c194de",
-        url: "https://github.com/zwilias/elm-json/releases/download/v0.2.8/elm-json-v0.2.8-x86_64-unknown-linux-musl.tar.gz",
-        fileSize: 2255688,
+      // Rosetta.
+      "darwin-arm64": {
+        hash: "1ca3cde58730e87f2b73afb2432d5d8c88787ee3745bc81769ec0eef962ceaf6",
+        url: "https://github.com/zwilias/elm-json/releases/download/v0.2.8/elm-json-v0.2.8-x86_64-apple-darwin.tar.gz",
+        fileSize: 838159,
         fileName: "elm-json",
         type: "tgz",
       },
-      mac: {
+      "darwin-x64": {
         hash: "1ca3cde58730e87f2b73afb2432d5d8c88787ee3745bc81769ec0eef962ceaf6",
         url: "https://github.com/zwilias/elm-json/releases/download/v0.2.8/elm-json-v0.2.8-x86_64-apple-darwin.tar.gz",
         fileSize: 838159,
         fileName: "elm-json",
         type: "tgz",
       },
-      windows: {
+      "linux-arm64": {
+        hash: "973405b81577889db391b7718d7bf7358e46be5cbecd4a7e72214f6388d2468e",
+        url: "https://github.com/zwilias/elm-json/releases/download/v0.2.8/elm-json-v0.2.8-armv7-unknown-linux-musleabihf.tar.gz",
+        fileSize: 2086957,
+        fileName: "elm-json",
+        type: "tgz",
+      },
```

Comment:
Need to check what’s correct here. See https://github.com/zwilias/elm-json/pull/51#discussion_r1063986522

Reply:
If it works on both ARM 32 and ARM 64, maybe it makes sense to exclude the 32 version as we don’t have 32 bit binaries for anything else. It could be added later easily if someone asks for it.

Reply:
I asked on Discord and it does work on ARM 64.

Reply:
I added support for ARM 32 as well, and added https://github.com/dmy/elm-raspberry-pi/releases. So all tools have an ARM 32 option now.